### ISSUE TITLE
Erro in opening a data explorer view for a simple csv resource 

### ISSUE THREAD

**Issue by miguelbgouveia** [ckan/ckan#3233]
### CKAN Version
CKAN v2.6.0.

### Expected behaviour
I have an instance of ckan with the recline_view plugin activated and with the ckanext_spatial installed and the geo_view as the default plugin for geographic view data. Then I create a resource that have csv data and with geographic information on it. Adding this resource the ckan system creates automatically a data explorer view. Opening this view I expect to see the csv data in a grid a in a map.

### Actual behavior
When I open the view I get the following error:
```

Module ckan.controllers.package:1568 in resource_view          view
>>  return h.rendered_resource_view(view, resource, package, embed=True)
Module ckan.lib.helpers:1954 in rendered_resource_view          view
>>  return literal(base.render(template, extra_vars=data_dict))
Module ckan.lib.base:174 in render          view
>>  return cached_template(template_name, render_template)
Module pylons.templating:249 in cached_template          view
>>  return render_func()
Module ckan.lib.base:128 in render_template          view
>>  return render_jinja2(template_name, globs)
Module ckan.lib.base:85 in render_jinja2          view
>>  return template.render(**extra_vars)
Module jinja2.environment:989 in render          view
>>  return self.environment.handle_exception(exc_info, True)
Module jinja2.environment:754 in handle_exception          view
>>  reraise(exc_type, exc_value, tb)
Module /home/ubuntu/workspace/ckan/default/src/ckan/ckanext/reclineview/theme/templates/recline_view.html:1 in top-level template code          view
>>  {% extends "base.html" %}
Module /home/ubuntu/workspace/ckan/default/src/ckanext-geoview/ckanext/geoview/templates/base.html:1 in top-level template code          view
>>  {% ckan_extends %}
Module /home/ubuntu/workspace/ckan/default/src/ckan/ckan/templates/base.html:103 in top-level template code          view
>>  {%- block page %}{% endblock -%}
Module /home/ubuntu/workspace/ckan/default/src/ckan/ckanext/reclineview/theme/templates/recline_view.html:5 in block "page"          view
>>  {% set map_config = h.get_map_config() or h.get_mapview_config() %}
Module jinja2.environment:412 in getattr          view
>>  return obj[attribute]
Module ckan.lib.helpers:62 in __getitem__          view
>>  key=key
HelperError: Helper 'get_mapview_config' has not been defined.

```
### What steps can be taken to reproduce the issue? 
To reproduce this error we just need to create simple csv resource and define in the configuration ckan file the following extensions:

```
ckan.plugins = stats text_view image_view recline_view datastore datapusher resource_proxy spatial_metadata spatial_query geo_view geojson_view wmts_view

ckan.views.default_views = image_view text_view recline_view pdf_view geo_view
```

**Comment by amercader** [comment]
@miguelbgouveia Good catch, this was introduced on #3200 (cc @jrods). Fix in #3235, can you confirm that fixes the issue for you?

**Comment by miguelbgouveia** [comment]
Yes, this fix resolve my problem.
